File: serverless-backend/CommonWallet/Models/Payment.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommonWallet.Models
{
	public class Payment
	{
        [JsonProperty("payment_id")]
        public int PaymentId { get; set; }

        [JsonProperty("payer_name")]
        public string PayerName { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("amount")]
        public double Amount { get; set; }

        [JsonProperty("payment_time")]
        public DateTime PaymentTime { get; set; }
    }
}

File: android-app/app/src/main/java/com/example/commonwallet/models/Payment.kt
package com.example.commonwallet.models

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime
import java.time.ZonedDateTime

@Entity
data class Payment(
    @SerializedName("payment_id") @PrimaryKey val paymentId: Int,
    @SerializedName("payer_name") val payerName: String,
    @SerializedName("payment_time") val paymentTime: ZonedDateTime,
    val amount: Double,
    val description: String
)
